code/edgebridge/build.gradle

```diff
@@ -118,7 +118,7 @@ publishing {
                 url = 'https://developer.adobe.com/client-sdks'
                 licenses {
                     license {
-                        name = 'Adobe Proprietary'
+                        name = 'The Apache License, Version 2.0'
```

Comment:
Should we include the `url` and `distribution` fields as well? 
Here's the same for the Edge extension. https://github.com/adobe/aepsdk-edge-android/blob/main/code/edge/build.gradle#L132